rebuild-amplify/rebuild-amplify.ts

```diff
@@ -0,0 +1,121 @@
+import { createObjectCsvWriter } from "csv-writer";
```

Comment:
maybe add comments of the pre-setup req. needs aws creds to be setup

Reply:
i think that's mostly true of this repo as a whole and not just this file in particular. maybe we should delegate this to another PR (iirc we have this in notion)

Reply:
Yea haha we definitely should

Maybe for purpose here, can add 1-2 lines comments first?

Reply:
created a readme file